src/sdk/main/include/impl/EntityIdHelper.h

```diff
@@ -109,14 +109,6 @@ template<typename T = uint64_t>
  */
 [[nodiscard]] std::string checksum(std::string_view address, const LedgerId& ledgerId);
 
-/**
- * Determine if the input byte array could represent a long-zero address.
- *
- * @param address The byte array to determine if it could be a long-zero address.
- * @return \c TRUE if the input byte array could represent a long-zero address, otherwise \c FALSE.
- */
-[[nodiscard]] bool isLongZeroAddress(const std::vector<std::byte>& address);
```

Comment:
What implications does this have of just removing it? Can we no longer tell the difference between a long-zero address vs. an EVM address?